trunk/SUAVE/Methods/Noise/Fidelity_One/Propeller/propeller_noise_frequency.py

```diff
@@ -0,0 +1,327 @@
+            const = - density*sound_speed*sound_speed*blades_number*np.sin(angle)*np.exp(1j*m*blades_number*((frequency_doppler*dist_obser/sound_speed)-(np.pi/2.0))) \
+                / (8.0*np.pi*(dist_axis/propeller_diameter)*convection_factor)        
+    
+      #      print "const = ", const
+            
+            #********************************************
+            #Loop over the blade geometry - sections
+            for i in range (0,number_sections):    
+            #********************************************
+                
+                max_thickness = 0.094*airfoil_chord[i]
+                z  = r0[i]/tip_radius                       #normalized radial coordinate
+                tb = max_thickness/airfoil_chord[i]         #section thickness to chord ratio
+                Bd = airfoil_chord[i]/propeller_diameter    #section chord to diameter ratio
+                
+                Mach_section = np.sqrt((Mx*Mx)+(z*z*Mach_tip*Mach_tip))
+            
+               
+                #Chordwise wave number
+                kx = (2.0*m*blades_number*Bd*Mach_tip)/(Mach_section*convection_factor)
+                
+                #Second wave number
+                ky = ((2*m*blades_number*Bd)/(z*Mach_section))*((Mach_section*Mach_section*np.cos(angle)-Mx)/convection_factor)
+                
+                #Phase shift due to sweep
+                phi_zero = (2*m*blades_number/z*Mach_section)*((Mach_section*Mach_section*np.cos(angle)-Mx)/convection_factor)*FA[i]/propeller_diameter
+                
+                #Phase shift due to offset or face alignment
+                phi_s = (2*m*blades_number*Mach_tip/Mach_section*convection_factor)*MCA[i]/propeller_diameter    
+        
+                
+                #Calculation of the Bessel function:
+                Arg    = m*blades_number*z*Mach_tip*np.sin(angle)/convection_factor #argument
+                NOrder = m*blades_number    #order
+                XJn    = scipy.special.jn(NOrder, Arg)
+                
+                if (kx==0):
+                    Gama_v = 2.0/3.0
+                    Gama_l = 1.0
+                    Gama_d = 1.0
+                else:
+                    Gama_v = (8.0/(kx*kx))*((2.0/kx)*np.sin(kx*0.5)-np.cos(kx*0.5))
+                    Gama_l = (2.0/kx)*np.sin(kx*0.5)
+                    Gama_d = (2.0/kx)*np.sin(kx*0.5)
+                
+                
+                pvm_factor = kx*kx*tb*Gama_v
+                pdm_factor = 1j*kx*(drag_coefficient[i]*0.5)*Gama_d
+                plm_factor = -1j*ky*(lift_coefficient[i]*0.5)*Gama_l
+                
+                const_integral = Mach_section*Mach_section*np.exp(1j*(phi_s+phi_zero))*XJn
+                
+                Pvm_section[i] = const_integral*pvm_factor 
+                Pdm_section[i] = const_integral*pdm_factor 
+                Plm_section[i] = const_integral*plm_factor
+                
+                #DIv_section[i] = XJn*Gama_v
```

Comment:
Remove comments

Reply:
I removed this file from this PR and will put it in a noise PR